js/interface.js

```diff
@@ -2111,6 +2111,30 @@ function getSubmissions() {
   return Fliplet.App.Submissions.get();
 }
 
+function setFirebaseStatus(credentialKey, origin) {
+  var environment;
```

Comment:
if (!credentials) return?

Reply:
@squallstar we calling this function only if have `submission.data['fl-credentials']` key. Is it possible that this key would be empty after we create a submission?

Reply:
@squallstar added a return statement to prevent code exec when we don't have the value of a credential.